src/main/target/SKYSTARSF722HD/target.h

```diff
@@ -17,9 +17,13 @@
 
 #pragma once
 
-#define TARGET_BOARD_IDENTIFIER "SS7D"
-
+#ifdef SKYSTARSF7MINIHD
+#define TARGET_BOARD_IDENTIFIER "SSF7MHD"
```

Comment:
Isn't TARGET_BOARD_IDENTIFIER supposed to be 4 characters only?

Reply:
I don't think it is a problem. Some target board identifiers are longer. I think the longest one is the one for the iFlight Blitz F7 Pro which is 12 chars long

Reply:
Perhaps we can use SS7D for one and SS7H for the other, to avoid issues with the TARGET_BOARD_IDENTIFIER.

Looking over the fence, on the BF side, it is just an identifier for the mcu used (makes sense for unified targets).

We can probably drop the uniquines requirement in INAV and change it to either be a 4 byte manufacturer id, or platform type like bf.

The TARGET_BOARD_IDENTIFIER is used in msp messages (limited to 4 bytes) and in crsf telemetry's device info. The crsf device info is just a string and could use USBD_PRODUCT_STRING instead.